src/mlpack/methods/decision_tree/decision_tree_impl.hpp

```diff
@@ -247,18 +366,26 @@ void DecisionTree<FitnessFunction,
   // numericAux and categoricalAux (and clear them later if we make not split),
   // and use classProbabilities as auxiliary information.  Later we'll overwrite
   // classProbabilities to the empirical class probabilities if we do not split.
-  double bestGain = FitnessFunction::Evaluate(labels, numClasses);
+
+  double bestGain = FitnessFunction::Evaluate(
+    labels.subvec(begin, begin + count - 1), numClasses);
```

Comment:
Minor comment---this should be indented twice since it is a continuation of a previous line.